rrat/static/css/input.css

```diff
@@ -1,3 +1,23 @@
+/* input.css */
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* Add any custom styles below this line */
+
+.hero-section {
+  @apply bg-indigo-900 text-white py-16; /* Example Tailwind classes */
+}
+
+.hero-heading {
+  @apply text-4xl font-bold mb-4; /* Example Tailwind classes */
+}
+
+.hero-subheading {
+  @apply text-lg mb-6; /* Example Tailwind classes */
+}
+
+.hero-button {
+  @apply bg-white text-indigo-900 px-6 py-3 rounded-full font-semibold hover:bg-indigo-100 transition;
+}
```

Comment:
remove this and apply the tailwind classes to the tags/elements in the home template